web-registry/src/components/atoms/Link.tsx

```diff
@@ -1,13 +1,9 @@
 import React from 'react';
+import { Link as MuiLink, LinkProps as MuiLinkProps } from '@mui/material';
 import { Link as RouterLink } from 'react-router-dom';
-import MuiLink from '@mui/material/Link';
-import { SxProps } from '@mui/system';
-import { Theme } from 'web-components/lib/theme/muiTheme';
-import { LinkItem } from 'web-components/lib/components/footer/footer-new';
 
-interface LinkProps extends LinkItem {
```

Comment:
I ran into issues passing this component to another which accepted a `Link` through props. This _could_ be slightly confusing because 

I'm wondering if `LinkItem` should be updated as well, but am leaving for now